src/components/Dashboard/MobileAppBanner/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { ReactElement, useCallback, useState } from 'react'
+import styled from 'styled-components'
+import { Button } from '@gnosis.pm/safe-react-components'
+import AppstoreButton from 'src/components/AppstoreButton'
+import { trackCustomClick } from 'src/utils/googleTagManager'
+import useCachedState from 'src/utils/storage/useCachedState'
+
+const StyledContainer = styled.div`
+  position: relative;
+  width: 480px;
+  height: 224px;
+`
+
+const StyledBanner = styled.div`
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-image: ${({ $count }: { $count: number }) => `url(./mobile-banner/${$count}.png)`};
+  transition: background-image 100ms linear;
+  cursor: pointer;
+  height: 100%;
+  width: 100%;
+`
+
+const StyledAppstoreButton = styled.div`
+  position: absolute;
+  z-index: 2;
+  left: 37px;
+  bottom: 27px;
+  opacity: 0;
+  cursor: pointer;
+`
+
+const StyledCloseButton = styled.button`
+  position: absolute;
+  z-index: 2;
+  right: 21px;
+  top: 22px;
+  opacity: 0.5;
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+  opacity: 0;
+`
+
+const StyledButtons = styled.div`
+  position: absolute;
+  z-index: 2;
+  left: 22px;
+  bottom: 30px;
+  cursor: pointer;
+  opacity: 0;
+`
+
+// Tracking
+const eventCategory = 'mobile-app-promotion'
+const eventAction = 'dashboard-banner'
+const eventLabels = {
+  alreadyUse: 'already-use',
+  notInterested: 'not-interested',
+}
```

Comment:
We should put these in the `src/utils/events` folder.

Reply:
The tracking API is not very user-friendly TBH.
All those generated events and enums 🤯 
I suggest we refactor it to be simpler and more verbose and explicit.

Reply:
I've added a type CustomEvent and moved the mobile events to utils.